qemu_v8.mk

```diff
@@ -65,7 +65,9 @@ arm-tf-clean:
 # QEMU
 ################################################################################
 qemu:
-	cd $(QEMU_PATH); ./configure --target-list=aarch64-softmmu\
+	cd $(QEMU_PATH); \
+		git submodule update --init dtc && \
```

Comment:
qemu should not create dtc/ if `git submodule` is not called? I can double-check this.